### PR TITLE
Added thoth-station/performance repository

### PR DESCRIPTION
**Pull request by pacospace**
Added thoth-station/performance repository so Sesheta can work on it.

Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "graph-sync-scheduler",
         "httpd-aicoe-container",
         "init-job",
-	"invectio",
+	    "invectio",
```
Comment (fridex) [inline config.json]: :eye:

Comment (pacospace) [reply config.json]: :laughing: it was not inline :smiley:

Comment (fridex) [reply config.json]: Tab versus spaces :angry: :wink:

Comment (pacospace) [reply config.json]: Should be fine now :laughing:

**Review by goern — APPROVED** [review]

**Comment by goern** [comment]
@harshad16 we need @sesheta here to put the approved label on all approved PR, otherwise, zuul will not merge.

**Comment by harshad16** [comment]
@goern We would need to deploy @sesheta for aicoe config https://raw.githubusercontent.com/AICoE/sesheta/master/config-aicoe.json as this repo is in different organizations.

**Comment by goern** [comment]
Let's do it ;)

**Pull request merged**